src/Scene.h

```diff
@@ -68,6 +72,12 @@ class CScene
 	bool intersect(Ray& ray) const
 	{
 		// --- PUT YOUR CODE HERE ---
+		bool hit = false;
+		for (auto pPrim : m_vpPrims) {
+			if (pPrim->intersect(ray)) {
+				hit = true;
+			}
+		}
 		return false;
```

Comment:
This is obviously wrong. You return false all the time - so nothing can be rendered! Did you even check any results?